packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEditForm.tsx

```diff
@@ -119,6 +119,8 @@ export const RequestLineEditForm = ({
             {t('heading.comment')}
           </Typography>
           <TextArea
+            value={draftLine?.comment}
+            onChange={e => update({ comment: e.target.value })}
```

Comment:
The comment component was already in place.. just needed to add it to the query and pass through the values/trigger an update